ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Exif.java

```diff
@@ -154,31 +192,40 @@ private TiffOutputField createTiffOutputField(@NonNull TiffField tiffField) {
         }
 
         @NonNull
-        public Builder setUserComment(boolean addMake, boolean addModel) {
-            addUserCommentStringExif(mExifDirectory, createUserComment(addMake, addModel));
+        public Builder setUserComment(String userComment) {
```

Comment:
Extracted User Comment creation into the `Exif.UserCommentBuilder` to easily add new values and enforce requirements for creating User Comment strings.